source/MaterialXRuntime/Private/PvtMessage.cpp

```diff
@@ -73,9 +73,6 @@ void PvtMessageHandler::removeCallback(RtCallbackId id)
     auto it = _callbackIdToType.find(id);
     if (it != _callbackIdToType.end())
     {
-        // Remove from the message type map
-        _callbackIdToType.erase(id);
```

Comment:
Fixes a command test crash. Basically an iterator was being used after a value was deleted making the iterator invalid. The change is to use the iterator first, then do the delete.